README.md

```diff
@@ -37,6 +37,19 @@ curl -s "https://raw.githubusercontent.com/Ph0enixKM/AmberNative/master/setup/in
 ```
 
 #### Via a package manager
+
+### Snap
```

Comment:
i don't like the language how it is written.

what's stopping from adding one line after "Amber is packages in the following distros"?

Reply:
like:

```
#### Via a package manager
Amber is packaged in the following distros:

Arch (AUR) - `amber-bash-bin`
Snap - `amber-bash`
```

Reply:
Because `snap` is not a distro specific package. Even though it's used mainly by Ubuntu and related distros, snap can be used in any snap supported distro. Essentially removing the need for thousands of other packages. Because this will work just like the native package.